lib/beaker-pe/install/pe_utils.rb

```diff
@@ -764,8 +764,10 @@ def feature_flag?(flag, opts)
         # @return [Boolean] true if frictionless is supported and not affected by known bugs
         def install_via_msi?(host)
           #windows agents from 4.0 -> 2016.1.2 were only installable via the aio method
-          #powershell2 bug was fixed in PE 2016.4.3
-          (host['platform'] =~ /windows/ && (version_is_less(host['pe_ver'], '2016.4.0') && !version_is_less(host['pe_ver'], '3.99'))) || (host['platform'] =~ /windows-2008r2/ && (version_is_less(host['pe_ver'], '2016.4.3') && !version_is_less(host['pe_ver'], '3.99')))
+          #powershell2 bug was fixed in PE 2016.4.3, and PE 2017.1.0, but not 2016.5.z.
+          (host['platform'] =~ /windows/ && (version_is_less(host['pe_ver'], '2016.4.0') && !version_is_less(host['pe_ver'], '3.99'))) ||
```

Comment:
I think something like that should be done in a major version bump / branch (to adhere to semver) - Customers do use beaker and beaker-pe for testing, and despite us EOL'ing it, some customers are still stuck on 3.x series.